src/package/cppwinrt/vsix/ProjectTemplates/VC/Windows Desktop/ConsoleApplication/ConsoleApplication.vcxproj

```diff
@@ -2,6 +2,7 @@
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Globals">
     <CppWinRTOptimized>true</CppWinRTOptimized>
+    <CppWinRTNamespaceAutoMerge>true</CppWinRTNamespaceAutoMerge>
```

Comment:
Suggestions for better names?

Reply:
name is fine.  would you mind adding an entry to the Customizing section in src/package/cppwinrt/nuget/readme.md?

Reply:
Sure, will add that.